netbeans/README.txt

```diff
@@ -0,0 +1,23 @@
+Steps to build and release the NetBeans Checker Framework plugin:
+1. Copy Checker Framework artifacts to the netbeans/CheckerFrameworkPlugin/release directory (just checker.jar and checker-qual.jar should be enough)
+2. Run the following command: ant -f checker-framework/netbeans/CheckerFrameworkPlugin nbm
```

Comment:
Also need to update the version number to the Checker Framework version.  Project properties -> API Versioning. Then change the "Specification Version".